### PR TITLE
Added youtube under community in footer

### LINKED ISSUE #119: Other:  Add Youtube link to new Uptane site footer
### Issue Summary

 Add the Uptane Youtube link to the new Uptane site footer, under the community section 
  
image

here is the link to the channel: https://www.youtube.com/@Uptane

### PR DESCRIPTION
**Pull request by tejas-raskar**
# Summary of Changes

Added the Youtube link of Uptane in the Community section of the footer

## Related Issue

Closes #119 

## Checklist

- [x] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
- [x] My code is well-documented, and I've updated relevant documentation.
- [ ] I have added or updated test cases to ensure the code's functionality.
- [x] I have tested these changes on my local environment.
- [x] All tests pass, and there are no new linting errors.
- [x] I have reviewed and proofread my code and the changes.
- [x] The branch is up-to-date with the base branch.

## Reviewer(s)

@Abhijay007

### CODE REVIEW

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Comment by tejas-raskar** [comment]
Hey @Abhijay007, can we add the hacktoberfest-accepted label before the deadline, so that it can be considered in hacktoberfest?

If any changes are required after the review I'll do them too

File: docusaurus.config.js [hunk 1]
```diff
@@ -192,6 +192,10 @@ const config = {
                 label: 'LinkedIn',
                 href: 'https://linkedin.com/company/uptane',
               },
+              {
+                label: 'Youtube',
+                href: 'https://www.youtube.com/@Uptane',
+              }
```
Comment (hexsecs) [inline docusaurus.config.js:198]:
```suggestion
              },
```
I think we want the trailing comma based on best practices.

**Review by hexsecs — CHANGES_REQUESTED** [review]
Everything else looks good.

**Comment by tejas-raskar** [comment]
The latest commit by @hexsecs adds the trailing comma, are there any further changes needed?

**Review by hexsecs — APPROVED** [review]

**Pull request merged**